### ISSUE TITLE
Event binding of template column is not working

### ISSUE THREAD

**Issue by vipintme** [aurelia-ui-toolkits/aurelia-syncfusion-bridge#59]
We have implemented implemented the grid using aurelia-syncfusion-bridge. We have added a template column for showing the edit button

```
`<ej-column e-field="userId" e-header-text="Edit" e-text-align="center" e-on-record-click.delegate="openUser()">
                    <ej-template>
                        <div><button class="btn btn-info"   click.delegate="openUser()"><span class="glyphicon glyphicon-pencil"></span></button> </div>
                    </ej-template>
                </ej-column>`
```

In the view model , We have defined the openUser() method. The event binding of  this button is not working.When cliking this button, we are getting error that "openUser is not a function". We are developed the application using aurelia+ webpack

**Comment by karthickthangasamy** [comment]
@vipintme confirmed this as bug and provide you the solution at the earliest.

**Comment by karthickthangasamy** [comment]
This issue fixed with latest `aurelia-syncfusion-bridge@0.8.0`
